### ISSUE TITLE
[Urgent] Rabit tracker.py is missing from xgboost4j-gpu_2.12 1.5.0 maven JAR

### ISSUE THREAD

**Issue by dbczumar** [dmlc/xgboost#7449]
While attempting to fit an `ml.dmlc.xgboost4j.scala.spark.XGBoostClassifier` with `ml.dmlc:xgboost4j-gpu_2.12:1.5.0` and `ml.dmlc:xgboost4j-spark-gpu_2.12:1.5.0` (https://mvnrepository.com/artifact/ml.dmlc/xgboost4j-gpu_2.12/1.5.0), I observed the following error:

```
21/11/19 05:34:35 INFO XGBoostClassifier: ml.dmlc.xgboost4j.scala.spark.XGBoostClassifier inferred 1 classes for labelCol=xgbc_0349f543fcdf__labelCol since numClasses was not specified in the column metadata.
21/11/19 05:34:35 INFO CodeGenerator: Code generated in 65.294317 ms
21/11/19 05:34:35 INFO XGBoostSpark: Running XGBoost 1.5.0 with parameters:
alpha -> 0.0
min_child_weight -> 1.0
sample_type -> uniform
base_score -> 0.5
rabit_timeout -> -1
colsample_bylevel -> 1.0
grow_policy -> depthwise
skip_drop -> 0.0
lambda_bias -> 0.0
silent -> 0
scale_pos_weight -> 1.0
seed -> 0
cache_training_set -> false
features_col -> features
num_early_stopping_rounds -> 0
label_col -> approved
num_workers -> 1
subsample -> 1.0
lambda -> 1.0
max_depth -> 2
probability_col -> probability
raw_prediction_col -> rawPrediction
kill_spark_context_on_worker_failure -> true
tree_limit -> 0
custom_eval -> null
dmlc_worker_connect_retry -> 5
rate_drop -> 0.0
max_bin -> 256
train_test_ratio -> 1.0
use_external_memory -> false
objective -> binary:logistic
eval_metric -> logloss
num_round -> 15
timeout_request_workers -> 1800000
missing -> NaN
rabit_ring_reduce_threshold -> 32768
checkpoint_path -> 
tracker_conf -> TrackerConf(0,python)
tree_method -> gpu_hist
max_delta_step -> 0.0
eta -> 0.3
verbosity -> 1
colsample_bytree -> 1.0
normalize_type -> tree
allow_non_zero_for_missing -> false
custom_obj -> null
gamma -> 0.0
sketch_eps -> 0.03
nthread -> 1
prediction_col -> prediction
checkpoint_interval -> -1
21/11/19 05:34:35 WARN XGBoostSpark: train_test_ratio is deprecated since XGBoost 0.82, we recommend to explicitly pass a training and multiple evaluation datasets by passing 'eval_sets' and 'eval_set_names'
21/11/19 05:34:35 ERROR RabitTracker: load tracker library failed.
21/11/19 05:34:35 ERROR RabitTracker: java.io.FileNotFoundException: File /tracker.py was not found inside JAR.
21/11/19 05:34:35 INFO RabitTracker$TrackerProcessLogger: python: can't open file 'null': [Errno 2] No such file or directory
21/11/19 05:34:35 INFO RabitTracker$TrackerProcessLogger: Tracker Process ends with exit code 2
21/11/19 05:34:35 INFO XGBoostSpark: starting training with timeout set as 1800000 ms for waiting for resources
21/11/19 05:34:35 INFO RabitTracker: Tracker Process ends with exit code 2
21/11/19 05:34:35 INFO XGBoostSpark: Rabit returns with exit code 2
21/11/19 05:34:35 ERROR XGBoostSpark: the job was aborted due to 
ml.dmlc.xgboost4j.java.XGBoostError: XGBoostModel training failed
	at ml.dmlc.xgboost4j.scala.spark.XGBoost$.postTrackerReturnProcessing(XGBoost.scala:750)
	at ml.dmlc.xgboost4j.scala.spark.XGBoost$.trainDistributed(XGBoost.scala:624)
	at ml.dmlc.xgboost4j.scala.spark.XGBoostClassifier.train(XGBoostClassifier.scala:199)
	at ml.dmlc.xgboost4j.scala.spark.XGBoostClassifier.train(XGBoostClassifier.scala:40)
	at org.apache.spark.ml.Predictor.fit(Predictor.scala:151)
	at org.apache.spark.ml.Predictor.fit(Predictor.scala:115)
	at org.apache.spark.ml.Pipeline.$anonfun$fit$5(Pipeline.scala:151)
	at org.apache.spark.ml.MLEvents.withFitEvent(events.scala:131)
	at org.apache.spark.ml.MLEvents.withFitEvent$(events.scala:124)
	at org.apache.spark.ml.util.Instrumentation.withFitEvent(Instrumentation.scala:43)
	at org.apache.spark.ml.Pipeline.$anonfun$fit$4(Pipeline.scala:151)
	at scala.collection.Iterator.foreach(Iterator.scala:943)
	at scala.collection.Iterator.foreach$(Iterator.scala:943)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
	at org.apache.spark.ml.Pipeline.$anonfun$fit$2(Pipeline.scala:147)
	at org.apache.spark.ml.MLEvents.withFitEvent(events.scala:131)
	at org.apache.spark.ml.MLEvents.withFitEvent$(events.scala:124)
	at org.apache.spark.ml.util.Instrumentation.withFitEvent(Instrumentation.scala:43)
	at org.apache.spark.ml.Pipeline.$anonfun$fit$1(Pipeline.scala:133)
	at org.apache.spark.ml.util.Instrumentation$.$anonfun$instrumented$1(Instrumentation.scala:284)
	at scala.util.Try$.apply(Try.scala:213)
	at org.apache.spark.ml.util.Instrumentation$.instrumented(Instrumentation.scala:284)
	at org.apache.spark.ml.Pipeline.fit(Pipeline.scala:133)
	at $line49c7dc07c68f459591bffad31c383db967.$read$$iw$$iw$$iw$$iw$$iw$$iw.<init>(command-1886180595162807:1)
	at $line49c7dc07c68f459591bffad31c383db967.$read$$iw$$iw$$iw$$iw$$iw.<init>(command-1886180595162807:48)
	at $line49c7dc07c68f459591bffad31c383db967.$read$$iw$$iw$$iw$$iw.<init>(command-1886180595162807:50)
	at $line49c7dc07c68f459591bffad31c383db967.$read$$iw$$iw$$iw.<init>(command-1886180595162807:52)
	at $line49c7dc07c68f459591bffad31c383db967.$read$$iw$$iw.<init>(command-1886180595162807:54)
	at $line49c7dc07c68f459591bffad31c383db967.$read$$iw.<init>(command-1886180595162807:56)
	at $line49c7dc07c68f459591bffad31c383db967.$read.<init>(command-1886180595162807:58)
	at $line49c7dc07c68f459591bffad31c383db967.$read$.<init>(command-1886180595162807:62)
	at $line49c7dc07c68f459591bffad31c383db967.$read$.<clinit>(command-1886180595162807)
	at $line49c7dc07c68f459591bffad31c383db967.$eval$.$print$lzycompute(<notebook>:7)
	at $line49c7dc07c68f459591bffad31c383db967.$eval$.$print(<notebook>:6)
	at $line49c7dc07c68f459591bffad31c383db967.$eval.$print(<notebook>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:747)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain.$anonfun$interpret$1(IMain.scala:568)
	at scala.reflect.internal.util.ScalaClassLoader.asContext(ScalaClassLoader.scala:36)
	at scala.reflect.internal.util.ScalaClassLoader.asContext$(ScalaClassLoader.scala:116)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:41)
	at scala.tools.nsc.interpreter.IMain.loadAndRunReq$1(IMain.scala:567)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:594)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:564)
	at com.databricks.backend.daemon.driver.DriverILoop.execute(DriverILoop.scala:219)
	at com.databricks.backend.daemon.driver.ScalaDriverLocal.$anonfun$repl$1(ScalaDriverLocal.scala:235)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at com.databricks.backend.daemon.driver.DriverLocal$TrapExitInternal$.trapExit(DriverLocal.scala:926)
	at com.databricks.backend.daemon.driver.DriverLocal$TrapExit$.apply(DriverLocal.scala:879)
	at com.databricks.backend.daemon.driver.ScalaDriverLocal.repl(ScalaDriverLocal.scala:235)
	at com.databricks.backend.daemon.driver.DriverLocal.$anonfun$execute$13(DriverLocal.scala:564)
	at com.databricks.logging.UsageLogging.$anonfun$withAttributionContext$1(UsageLogging.scala:215)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:62)
	at com.databricks.logging.AttributionContext$.withValue(AttributionContext.scala:95)
	at com.databricks.logging.UsageLogging.withAttributionContext(UsageLogging.scala:213)
	at com.databricks.logging.UsageLogging.withAttributionContext$(UsageLogging.scala:210)
	at com.databricks.backend.daemon.driver.DriverLocal.withAttributionContext(DriverLocal.scala:50)
	at com.databricks.logging.UsageLogging.withAttributionTags(UsageLogging.scala:251)
	at com.databricks.logging.UsageLogging.withAttributionTags$(UsageLogging.scala:243)
	at com.databricks.backend.daemon.driver.DriverLocal.withAttributionTags(DriverLocal.scala:50)
	at com.databricks.backend.daemon.driver.DriverLocal.execute(DriverLocal.scala:541)
	at com.databricks.backend.daemon.driver.DriverWrapper.$anonfun$tryExecutingCommand$1(DriverWrapper.scala:693)
	at scala.util.Try$.apply(Try.scala:213)
	at com.databricks.backend.daemon.driver.DriverWrapper.tryExecutingCommand(DriverWrapper.scala:685)
	at com.databricks.backend.daemon.driver.DriverWrapper.getCommandOutputAndError(DriverWrapper.scala:526)
	at com.databricks.backend.daemon.driver.DriverWrapper.executeCommand(DriverWrapper.scala:638)
	at com.databricks.backend.daemon.driver.DriverWrapper.runInnerLoop(DriverWrapper.scala:431)
	at com.databricks.backend.daemon.driver.DriverWrapper.runInner(DriverWrapper.scala:374)
	at com.databricks.backend.daemon.driver.DriverWrapper.run(DriverWrapper.scala:225)
	at java.lang.Thread.run(Thread.java:748)
21/11/19 05:34:35 INFO HiveServer2: Shutting down HiveServer2
21/11/19 05:34:35 INFO ThriftCLIService: Caught InterruptedException. Shutting down thrift server.
21/11/19 05:34:35 INFO AbstractConnector: Stopped ServerConnector@21a462ce{SSL, (ssl, http/1.1)}{0.0.0.0:10000}
21/11/19 05:34:35 INFO session: node0 Stopped scavenging
21/11/19 05:34:35 INFO AbstractConnector: Stopped Spark@5f79b206{HTTP/1.1, (http/1.1)}{10.68.151.248:46359}
21/11/19 05:34:35 INFO ContextHandler: Stopped o.e.j.s.ServletContextHandler@18174bd9{/,null,STOPPED}
21/11/19 05:34:35 INFO ThriftCLIService: Thrift HTTP server has been stopped
21/11/19 05:34:35 INFO AbstractService: Service:ThriftHttpCLIService is stopped.
21/11/19 05:34:35 INFO AbstractService: Service:OperationManager is stopped.
21/11/19 05:34:35 INFO AbstractService: Service:SessionManager is stopped.
21/11/19 05:34:35 INFO AbstractService: Service:CLIService is stopped.
21/11/19 05:34:35 INFO AbstractService: Service:HiveServer2 is stopped.
21/11/19 05:34:35 INFO SparkUI: Stopped Spark web UI at http://10.68.151.248:46359
21/11/19 05:34:35 INFO StandaloneSchedulerBackend: Shutting down all executors
21/11/19 05:34:35 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
21/11/19 05:34:35 ERROR RabitTracker: Uncaught exception thrown by worker:
java.lang.IllegalStateException: SparkContext has been shutdown
	at org.apache.spark.SparkContext.runJobInternal(SparkContext.scala:2486)
	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2477)
	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2515)
	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2534)
	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2559)
	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$1(RDD.scala:1025)
	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:165)
	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:125)
	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:112)
	at org.apache.spark.rdd.RDD.withScope(RDD.scala:419)
	at org.apache.spark.rdd.RDD.foreachPartition(RDD.scala:1023)
	at ml.dmlc.xgboost4j.scala.spark.XGBoost$$anon$1.run(XGBoost.scala:612)
21/11/19 05:34:35 ERROR RabitTracker: java.lang.InterruptedException: sleep interrupted
```

Downloading and inspecting the JAR contents from Maven indicates that `tracker.py` is missing from `ml.dmlc:xgboost4j-gpu_2.12:1.5.0` entirely. Perhaps some build steps were missed? (e.g. similar to https://github.com/dmlc/xgboost/issues/1294#issuecomment-228334854).

```
$ mvn dependency:get -Dartifact=ml.dmlc:xgboost4j-gpu_2.12:1.5.0
corey.zumar@C02F17H3MD6R ~/.m2/repository/ml/dmlc/xgboost4j-gpu_2.12/1.5.0 $ jar -tf xgboost4j-gpu_2.12-1.5.0.jar | grep tracker
(base) corey.zumar@C02F17H3MD6R ~/.m2/repository/ml/dmlc/xgboost4j-gpu_2.12/1.5.0 $
```

In contrast, this file appears to be present in the CPU-only version (`ml.dmlc:xgboost4j_2.12:1.5.0`):

```
$ mvn dependency:get -Dartifact=ml.dmlc:xgboost4j_2.12:1.5.0
(base) corey.zumar@C02F17H3MD6R ~/.m2/repository/ml/dmlc/xgboost4j_2.12/1.5.0 $ jar -tf xgboost4j_2.12-1.5.0.jar | grep tracker
tracker.py
(base) corey.zumar@C02F17H3MD6R ~/.m2/repository/ml/dmlc/xgboost4j_2.12/1.5.0 $
```

For reference, here's the full contents of the impacted `xgboost4j-gpu_2.12-1.5.0.jar`:

```
(base) corey.zumar@C02F17H3MD6R ~/.m2/repository/ml/dmlc/xgboost4j-gpu_2.12/1.5.0 $ jar -tf xgboost4j-gpu_2.12-1.5.0.jar
META-INF/MANIFEST.MF
META-INF/
ml/
ml/dmlc/
ml/dmlc/xgboost4j/
ml/dmlc/xgboost4j/gpu/
ml/dmlc/xgboost4j/gpu/java/
ml/dmlc/xgboost4j/java/
ml/dmlc/xgboost4j/java/util/
ml/dmlc/xgboost4j/scala/
ml/dmlc/xgboost4j/scala/rabit/
ml/dmlc/xgboost4j/scala/rabit/handler/
ml/dmlc/xgboost4j/scala/rabit/util/
lib/
lib/linux/
lib/linux/x86_64/
META-INF/maven/
META-INF/maven/ml.dmlc/
META-INF/maven/ml.dmlc/xgboost4j-gpu_2.12/
xgboost4j-version.properties
ml/dmlc/xgboost4j/java/Rabit.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$$anonfun$4.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$AwaitingConnections$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$SetupComplete$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$StartTracker.class
ml/dmlc/xgboost4j/LabeledPoint.class
ml/dmlc/xgboost4j/gpu/java/CudfUtils$1.class
ml/dmlc/xgboost4j/java/GpuTestSuite.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$DataStruct$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$RabitWorkerRequest.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$ReduceWaitCount.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$IntraWorkerMessage.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerRecover$.class
ml/dmlc/xgboost4j/scala/rabit/RabitTracker$.class
ml/dmlc/xgboost4j/gpu/java/CudfUtils.class
ml/dmlc/xgboost4j/java/XGBoostError.class
ml/dmlc/xgboost4j/java/RabitTracker$1.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$$anonfun$6.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$TrackerCommand.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerStart.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerRecover.class
ml/dmlc/xgboost4j/scala/rabit/util/AssignedRank$.class
ml/dmlc/xgboost4j/gpu/java/CudfColumnBatch.class
ml/dmlc/xgboost4j/java/RabitTracker$TrackerProcessLogger.class
ml/dmlc/xgboost4j/java/IRabitTracker$TrackerStatus.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$IntSeqField$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$TrackerControlMessage.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$ReduceWaitCount$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler.class
ml/dmlc/xgboost4j/scala/rabit/util/RabitTrackerHelpers.class
META-INF/maven/ml.dmlc/xgboost4j-gpu_2.12/pom.properties
ml/dmlc/xgboost4j/gpu/java/CudfColumn.class
ml/dmlc/xgboost4j/java/DMatrix$SparseType.class
ml/dmlc/xgboost4j/java/ExternalCheckpointManager.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$AcknowledgeAcceptance$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$InterruptTracker$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$RequestAwaitConnWorkers$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$.class
ml/dmlc/xgboost4j/scala/rabit/util/RabitTrackerHelpers$ByteBufferHelpers.class
ml/dmlc/xgboost4j/gpu/java/CudfUtils$Builder.class
ml/dmlc/xgboost4j/java/IEvaluation.class
ml/dmlc/xgboost4j/java/XGBoost$CVPack.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$$anonfun$receive$1.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$DropFromWaitingList$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$$anonfun$3.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$AwaitingErrorCount$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$DivulgedWorkerHostPort$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$InterruptTracker.class
META-INF/maven/ml.dmlc/xgboost4j-gpu_2.12/pom.xml
ml/dmlc/xgboost4j/java/Booster.class
ml/dmlc/xgboost4j/java/NativeLibLoader$OS.class
ml/dmlc/xgboost4j/scala/XGBoost.class
ml/dmlc/xgboost4j/scala/XGBoost$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$RequestCompletionFuture$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$StringField$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$DataStruct.class
ml/dmlc/xgboost4j/scala/ExternalCheckpointParams.class
ml/dmlc/xgboost4j/java/XGBoost.class
ml/dmlc/xgboost4j/java/Column.class
ml/dmlc/xgboost4j/java/RabitTracker.class
ml/dmlc/xgboost4j/scala/ObjectiveTrait.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$$anonfun$7.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$DropFromWaitingList.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerStarted.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerShutdown$.class
ml/dmlc/xgboost4j/LabeledPoint$.class
ml/dmlc/xgboost4j/java/DeviceQuantileDMatrix.class
ml/dmlc/xgboost4j/java/XGBoostJNI.class
ml/dmlc/xgboost4j/java/IRabitTracker.class
ml/dmlc/xgboost4j/java/IObjective.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$$anonfun$1.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerTrackerPrint$.class
ml/dmlc/xgboost4j/scala/rabit/handler/WorkerDependencyResolver$Fulfillment.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$$anonfun$8.class
ml/dmlc/xgboost4j/scala/DMatrix.class
ml/dmlc/xgboost4j/java/Rabit$OpType.class
ml/dmlc/xgboost4j/java/DataBatch$BatchIterator.class
ml/dmlc/xgboost4j/scala/EvalTrait.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$State.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$AcknowledgeAcceptance.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerTrackerPrint.class
ml/dmlc/xgboost4j/scala/rabit/util/LinkMap.class
ml/dmlc/xgboost4j/java/NativeLibLoader$Arch.class
ml/dmlc/xgboost4j/scala/Booster.class
ml/dmlc/xgboost4j/scala/rabit/handler/WorkerDependencyResolver$$anonfun$receive$2.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerShutdown.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$AwaitingPortNumber$.class
ml/dmlc/xgboost4j/scala/ExternalCheckpointManager.class
ml/dmlc/xgboost4j/java/NativeLibLoader.class
ml/dmlc/xgboost4j/scala/DMatrix$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$IntField$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$RequestAwaitConnWorkers.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$DivulgedWorkerHostPort.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$RequestBoundFuture$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$.class
lib/linux/x86_64/libxgboost4j.so
ml/dmlc/xgboost4j/java/util/BigDenseMatrix.class
ml/dmlc/xgboost4j/scala/ExternalCheckpointParams$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$$anonfun$2.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$RabitWorkerResponse.class
ml/dmlc/xgboost4j/scala/rabit/util/AssignedRank.class
ml/dmlc/xgboost4j/java/util/UtilUnsafe.class
ml/dmlc/xgboost4j/java/ColumnBatch.class
ml/dmlc/xgboost4j/java/DMatrix.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerStarted$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$AwaitingConnections.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitTrackerHandler$StartTracker$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$WorkerStart$.class
ml/dmlc/xgboost4j/scala/rabit/util/RabitTrackerHelpers$.class
ml/dmlc/xgboost4j/java/Rabit$DataType.class
ml/dmlc/xgboost4j/java/DataBatch.class
ml/dmlc/xgboost4j/java/NativeLibLoader$1.class
ml/dmlc/xgboost4j/scala/rabit/handler/WorkerDependencyResolver.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$RequestWorkerHostPort$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$AwaitingHandshake$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$BuildingLinkMap$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$AwaitingCommand$.class
ml/dmlc/xgboost4j/scala/rabit/RabitTracker.class
ml/dmlc/xgboost4j/java/Booster$FeatureImportanceType.class
ml/dmlc/xgboost4j/java/TrackerProperties.class
ml/dmlc/xgboost4j/scala/rabit/handler/WorkerDependencyResolver$Fulfillment$.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$$anonfun$5.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler$DataField.class
ml/dmlc/xgboost4j/scala/rabit/handler/RabitWorkerHandler.class
ml/dmlc/xgboost4j/scala/rabit/util/RabitTrackerHelpers$ByteStringHelplers.class
```

**Comment by hcho3** [comment]
@dbczumar My bad. I will publish a new patch version to include  the missing `tracker.py`.

@trivialfis Can we wrap up the 1.5.1 version so that we can publish the same 1.5.1 version to both PyPI and Maven Central?

**Comment by dbczumar** [comment]
Thanks @hcho3 !

**Comment by trivialfis** [comment]
Sure, @hcho3 https://github.com/dmlc/xgboost/pull/7451 is the one last backport to 1.5.1 .

**Comment by trivialfis** [comment]
@hcho3 I will handle the Python release after R package for 1.5.0.2 is done.  Feel free to submit jvm packages to maven while you are at it.

**Comment by candalfigomoro** [comment]
@trivialfis @hcho3
As a side note, is there any known issue preventing the Scala tracker from being the default instead of the Python tracker for xgboost4j-spark?

**Comment by trivialfis** [comment]
> As a side note, is there any known issue preventing the Scala tracker from being the default instead of the Python tracker for xgboost4j-spark?

Lack of maintainer. ;-)

**Comment by hcho3** [comment]
@dbczumar I just published 1.5.1 version of XGBoost. It contains the tracker file.

**Comment by dbczumar** [comment]
Thank you, @hcho3 !
